TheMovieDB/SynopsisView.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  SwiftUIView.swift
+//  TheMovieDB
+//
+//  Created by Intern on 12/07/23.
+//
+
+import SwiftUI
+import SDWebImageSwiftUI
+
+struct SynopsisView: View {
+    var sampleMovie: Movies
```

Comment:
use @Binding, usually if you pass data to antoher view use binding, because lets say in future you have edit capability then you need binding to reflect data change to the viewmodel or your source of truth